Userland/killall.cpp

```diff
@@ -0,0 +1,95 @@
+#include <unistd.h>
+#include <stdio.h>
+#include <signal.h>
+#include <stdlib.h>
+#include <LibCore/CFile.h>
+#include <AK/AKString.h>
+
+static unsigned parse_uint(const String& str, bool& ok)
```

Comment:
We should use String::to_uint() instead of adding a separate parse_uint() function.